src/daft-functions/src/list/contains.rs

```diff
@@ -0,0 +1,71 @@
+use common_error::{DaftError, DaftResult};
+use daft_core::prelude::{DataType, Field, Schema, Series};
+use daft_dsl::{
+    functions::{ScalarFunction, ScalarUDF},
+    python::PyExpr,
+    ExprRef,
+};
+use pyo3::{pyfunction, PyResult};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
+struct ListContainsFunction;
+
+#[typetag::serde]
+impl ScalarUDF for ListContainsFunction {
+    fn as_any(&self) -> &dyn std::any::Any {
+        self
+    }
+
+    fn name(&self) -> &'static str {
+        "list_contains"
+    }
+
+    fn to_field(&self, inputs: &[ExprRef], schema: &Schema) -> DaftResult<Field> {
+        match inputs {
+            [data, value] => {
+                let data_field = data.to_field(schema)?;
+                let value_field = value.to_field(schema)?;
+                match &data_field.dtype {
+                    DataType::List(inner_type) => {
```

Comment:
Another check you should do with `inner_type` is to check if it is a type that you can check equality with the values.

Reply:
Also, the dtype can be a `FixedSizeList`